### PR TITLE
4" Casters recipe

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
```SUMMARY: Content "Makes 4" Casters craftable"```

#### Purpose of change

Casters seem to be something doable without industrial equipement.
#### Describe the solution

Adding a recipe for 4" Casters.
#### Additional context

The recipe is in `/json/recipes/other/parts.json` I don't know if it should have been in `/json/recipes/recipe_vehicle.json` or in `/json/recipes/other/vehicle.json` instead.

Right now it takes : 
- some steel and some plastic
- Tools for molding the wheel, assuming it's full plastic, tools for cutting and welding the bits of metal around the wheel and screw driver and bolt turning to put the wheel in place.
- Fabrication 5
- 1h30

What is actually in the recipe can be adjusted of course.

### CODE REVIEW

**Comment by Fris0uman** [comment]
Should I remove "reversible" line instead of setting it to false ? Or is it something else entierly ?
``` json 
 DEBUG    : line 23:18: assignment does not update value


    "difficulty": 5,
    "time": 90000,
    "reversible":
                 ^
                  false,
10    "autolearn": true,
10    "using": [ [ "welding_standard", 4 ], [ "steel_tiny", 4 ] ],
10
 
 FUNCTION : void report_strict_violation(JsonObject&, const string&, const string&)
 FILE     : src/assign.h
 LINE     : 46

**Comment by fungusakafungus** [comment]
> Should I remove "reversible" line instead of setting it to false ? Or is it something else entierly ?

Just remove the line. I found `"reversible": "false"` nowhere else in the code, so it probably defaults to false and so the false assignment is superfluous.

**Comment by Fris0uman** [comment]
Yep that does it, thanks.

File: data/json/recipes/other/parts.json [hunk 1]
```diff
@@ -11,5 +11,19 @@
     "autolearn": true,
     "using": [ [ "welding_standard", 6 ] ],
     "components": [ [ [ "material_aluminium_ingot", 8 ] ] ]
+  },
+  {
+    "result": "wheel_caster",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_PARTS",
```
Comment (mlangsdorf) [inline data/json/recipes/other/parts.json]:
```suggestion
    "subcategory": "CSC_OTHER_VEHICLE",
```

**Comment by Fris0uman** [comment]
Done, but what's the rule on what goes where here ?

**Comment by mlangsdorf** [comment]
There's no strictly enforced rule, but I am trying to get new vehicle part recipes added to recipes_vehicle.json.

**Pull request merged**